infra/partner-mainnet/scripts/example.txt

```diff
@@ -0,0 +1,4 @@
+multichain-account-sk-mainnet-0=foo
+multichain-cipher-sk-mainnet-0=bar
+multichain-sign-sk-mainnet-0=baz
+multichain-sk-share-mainnet-0=qux
```

Comment:
let's do `multichain-sk-share-mainnet-0=1` here

Reply:
I just have this as an example, ideally this file will get deleted after they are uploaded to SM and we have everything working.